### PR TITLE
[Release Tooling] Update zip integration instructions for privacy manifest support

### PR DESCRIPTION
**Pull request by ncooke3**

### CODE REVIEW

File: FirebaseCore/CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,11 @@
 - Fixed validation issues in Xcode 15.3 that affected binary distributions
   including Analytics, Firestore (SwiftPM binary distribution), and the
   Firebase zip distribution. (#12441)
+- [Zip Distribution] The manual integration instructions found in the
+  `Firebase.zip` have been updated for Xcode 15 users. The updated instructions
```
Comment (paulb777) [inline FirebaseCore/CHANGELOG.md:15]: Is there anything different between Xcode 15.3 and earlier Xcode 15 versions?

File: ReleaseTooling/Template/README.md [hunk 2]
```diff
@@ -34,17 +34,18 @@ To integrate a Firebase SDK with your app:
...
    has a checkmark next to it, and that you've selected "Copy items if needed."
 
-   > ⚠ Do not add the Firebase frameworks to the **Embed Frameworks** Xcode build
-   > phase. The Firebase frameworks are not embedded dynamic frameworks, but are
-   > [static frameworks](https://www.raywenderlich.com/65964/create-a-framework-for-ios)
-   > which cannot be embedded into your application's bundle.
+7. If using Xcode 15, embed each framework that was dragged in. Navigate to the
+   target's _General_ settings and find _Frameworks, Libraries, & Embedded
+   Content_. For each framework dragged in from the `Firebase.zip`, select
+   **Embed & Sign**. This step will enable privacy manifests to be picked up by
```
Comment (ncooke3) [inline ReleaseTooling/Template/README.md:40]: > For each framework dragged in from the `Firebase.zip`, select
   **Embed & Sign**.

I'm opting to recommend to embed and sign **each** framework, regardless if it contains a privacy manifest. The alternative way would be to only embed and sign SDKs that have privacy manifests, but encoding that in the zip instructions may add more complexity that it's worth. Thoughts?

File: ReleaseTooling/Template/README.md [hunk 3]
```diff
@@ -53,21 +54,21 @@ To integrate a Firebase SDK with your app:
...
+12. If your app does not include any Swift implementation, you may need to add
    a dummy Swift file to the app to prevent Swift system library missing
    symbol linker errors. See
    https://forums.swift.org/t/using-binary-swift-sdks-from-non-swift-apps/55989.
 
    > ⚠ If prompted with the option to create a corresponding bridging header
    > for the new Swift file, select **Don't create**.
 
-12. You're done! Compile your target and start using Firebase.
+13. You're done! Compile your target and start using Firebase.
```
Comment (paulb777) [inline ReleaseTooling/Template/README.md:71]: s/Compile/Build/

**Review by paulb777 — APPROVED** [review]

File: ReleaseTooling/Template/README.md (hunk 3 above)
Comment (ncooke3) [reply ReleaseTooling/Template/README.md]: Done.

File: ReleaseTooling/Template/README.md (hunk 2 above)
Comment (paulb777) [reply ReleaseTooling/Template/README.md:40]: Agreed, and eventually they may all have manifests anyway.

File: FirebaseCore/CHANGELOG.md (hunk 1 above)
Comment (ncooke3) [reply FirebaseCore/CHANGELOG.md:15]: To test, I tried validating an app with all SDKs in Xcode 15.1, and it was successful.

Comment (ncooke3) [reply FirebaseCore/CHANGELOG.md:15]: I will merge this as is.

**Pull request merged**